src/lambda_function/s3_to_glue/app.py

```diff
@@ -0,0 +1,87 @@
+import os
+import json
+import boto3
```

Comment:
Nit: Take a look at this in terms of ordering your imports: https://peps.python.org/pep-0008/#imports.  There are vscode modules like `isort` that will do it for you.

`black` will auto format your code for you to provide a structure.  That is what I use to format my code.